deliverables/README.md

```diff
@@ -2,13 +2,17 @@
 #### Required
 ##### Questions
 - What is a framework?
+ Like many web frameworks, Ruby on Rails uses the model–view–controller (MVC) pattern to organize application programming. In a  default configuration, a model in the Ruby on Rails framework maps to a table in a database, and a Ruby file.
 - What's the difference between a programming language and a framework?
+ A language is syntax, grammar, semantics.. that implementers are required to support. A framework is a cohesive set of -library  code that together simplifies programming in any given language.
 - What’s the difference between a library and a framework?
+ A library is a set of objects and functions that is focused around solving a particular problem or around a specific area of    application development (i.e. database access); a framework on the other hand is a collection of libraries centered around a    particular methodology (i.e. MVC) and covers all areas of application development.
```

Comment:
link to source:  http://stackoverflow.com/questions/148747/what-is-the-difference-between-a-framework-and-a-library